pkg/models/devops/devops.go

```diff
@@ -173,6 +173,8 @@ func (d devopsOperator) CreateDevOpsProject(workspace string, project *v1alpha3.
 	}
 	project.Name = ""
 	project.Labels[tenantv1alpha1.WorkspaceLabel] = workspace
+	project.Annotations[devopsv1alpha3.DevOpeProjectSyncStatusAnnoKey] = StatusPending
+	project.Annotations[devopsv1alpha3.DevOpeProjectSyncTimeAnnoKey] = GetSyncNowTime()
```

Comment:
Please consider putting line 176 and line 177 into a function instead of writing them four times.

Reply:
It is not a good idea to optimize in advance. There's only one line of code.